data_diff/database.py

```diff
@@ -188,11 +190,27 @@ def to_string(self, s: str):
 
 
 class Presto(Database):
-    def __init__(self, host, port, database, user, password):
+    def __init__(self, host, port, user, password, schema, database, print_sql=True):
```

Comment:
maybe we can update this signature to pass in "extra" args, then pass those extra args to `prestodb.dbapi.connect` might be a lighter weight change.

Reply:
I think excess kw should just be passed straight to the driver

Reply:
This code is out-dated